src/llmq/commitment.h

```diff
@@ -22,11 +22,13 @@ class CFinalCommitment
 {
 public:
     static constexpr uint16_t CURRENT_VERSION = 1;
+    static constexpr uint16_t INDEXED_QUORUM_VERSION = 2;
 
 public:
     uint16_t nVersion{CURRENT_VERSION};
     Consensus::LLMQType llmqType{Consensus::LLMQType::LLMQ_NONE};
     uint256 quorumHash;
+    int quorumIndex;
```

Comment:
@ogabrielides @PastaPastaPasta As discussed on slack, I think it would make sense to use a smaller size for quorum index. I can't imagine needing a full 32 bits to represent the number of active quorums for a specific quorum type. 16 bits seems reasonable to me (8 bits maybe, but I think I'd prefer 16 for plenty of room for future unknowns). @QuantumExplorer what do you think?

Reply:
@thephez @PastaPastaPasta 16 decidedby @QuantumExplorer